tests/units/typing/tensor/test_tensor.py

```diff
@@ -52,6 +51,60 @@ def test_unwrap():
     assert (ndarray == np.zeros((3, 224, 224))).all()
 
 
+def test_ellipsis_in_shape():
+    # ellipsis in the end, two extra dimensions needed
+    tensor = parse_obj_as(NdArray[3, ...], np.zeros((3, 128, 224)))
+    assert isinstance(tensor, NdArray)
+    assert isinstance(tensor, np.ndarray)
+    assert tensor.shape == (3, 128, 224)
+
+    # ellipsis in the end, no extra dimensions needed
+    tensor = parse_obj_as(NdArray[3, 128, 224, ...], np.zeros((3, 128, 224)))
```

Comment:
I am not sure, should not this one fail instead?

Reply:
yeah, makes sense to expect at least 1 dimension when users type ellipsis